src/core/lib/gprpp/ref_counted.h

```diff
@@ -111,7 +180,7 @@ class RefCounted : public Impl {
  protected:
   GPRC_ALLOW_CLASS_TO_USE_NON_PUBLIC_DELETE
 
-  RefCounted() { gpr_ref_init(&refs_, 1); }
+  RefCounted() = default;
```

Comment:
Should we start marking constructors as `constexpr` when possible? We haven't been doing it yet but I'm leaning that way for the future.

Reply:
Wait, isn't the std::atomic constructor constexpr when used with a default value? cppreference seems to suggest that it is. And, oh, sorry about not realizing that this is a derived class. Yeah, that makes it irrelevant.

Reply:
The ctor is constexpr but the actual ops wouldn't be `constexpr`. Oh I see. Sorry, now I understand. You were just referring to ctor. Sure, made the ctor contexpr.